src/integrations/Axeptio/loader.js

```diff
@@ -0,0 +1,18 @@
+import { LOAD_ORIGIN } from '../../utils/ScriptLoader';
+
+/* eslint-disable */
+// START-NO-SONAR-SCAN
+function loader(d, s, clientId) {
```

Comment:
`loadNativeSdk` is a better name. If you change the function names, ensure to rename it in Braze also for consistency.

Reply:
Updated